test/geometry/test_mesh_conversion.cpp

```diff
@@ -90,7 +92,7 @@ class TestMeshConversion : public ::testing::Test
       }
 
       // Faces
-      std::vector <std::uint32_t> face;
+      std::vector <index_t> face;
```

Comment:
I'd recommend you to make the ABI breaking PR into 2 parts:
* Add a `using Faces = std::vector<index_t>;` or similar alias
* Break ABI behind alias

Reply:
Wouldn't you say this should be just `Indices`?

Reply:
I don't know, haven't seen the code. It might be ID of face instead of being an index into a vector in which case having a different alias is helpful. Eg:
```cpp
struct vec4 { float x, y, z, w; };
using quaternion = vec4;
```